packages/rn-tester/js/utils/index.js

```diff
@@ -0,0 +1,142 @@
+/**
```

Comment:
Can we rename this file, with Haste every file name needs to be unique (across all of facebook). Something like `testerStateUtils`